gwt/pom.xml

```diff
@@ -40,39 +40,10 @@
       <classifier>sources</classifier>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <classifier>sources</classifier>
```

Comment:
Or is that included transitively via the dependency on `com.google.dagger:dagger`?